src/app/container/container.component.css

```diff
@@ -0,0 +1,20 @@
+/*
+*    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
The `WITHOUT WARRANTIES` bit seems extra, although true I guess. Might be an IDE setting, might want to grab something more standard like the license on https://github.com/dockstore/dockstore/blob/4b59cbb17be115288529314d817aed3aed72829d/dockstore-webservice/src/test/java/io/dockstore/webservice/resources/proposedGA4GH/package-info.java